e2e-tests/upgrade-consistency/run

```diff
@@ -33,7 +31,6 @@ main() {
 	compare_kubectl statefulset/${CLUSTER}-rs0 "-1110"
 
 	# test 1.12.0
-	API="psmdb.percona.com/v1-12-0"
```

Comment:
I am not sure about it because we need to test user's behavior. So, the user has 'psmdb.percona.com/v1-12-0' in CR. What should be done during the update? As I understand if I want to update my DB cluster to v1.13.0 , I need to change  'psmdb.percona.com/v1-12-0'  to  'psmdb.percona.com/v1' and set `crVersion` to '1.13.0'. Am I right?

Reply:
even if you set `apiVersion: psmdb.percona.com/v1-12-0`, Kubernetes sees it as `apiVersion: psmdb.percona.com/v1`. We were only doing that for kubectl annotation and now operator doesn't do that. You don't need to update `apiVersion` at all.